src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -127,13 +129,20 @@
                          (fn []
                            (rf/dispatch [:wallet/clean-scanned-address])
                            (rf/dispatch [:wallet/clean-local-suggestions]))))
-        [rn/scroll-view
-         {:content-container-style      style/container
-          :keyboard-should-persist-taps :handled
-          :scroll-enabled               false}
-         [account-switcher/view
-          {:on-press      on-close
-           :switcher-type :select-account}]
+        [floating-button-page/view
```

Comment:
@J-Son89 it is working fine. Just one thing, why are we not using insets from navigation instead of having to apply margin-top `https://github.com/status-im/status-mobile/pull/18388/files#diff-1fc8418ccf535f4dd924ee5e62463e5359e24b383c052afc4ab39d40c645d465R22-R24`